tests/mobilebert/test_tokenization_mobilebert.py

```diff
@@ -0,0 +1,341 @@
+# coding=utf-8
+# Copyright 2022 Leon Derczynski. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the MobileBERT tokenizer. """
+
+
+
+
+import os
+import unittest
+
+from transformers import BertTokenizerFast
+from transformers.models.bert.tokenization_bert import (
+    VOCAB_FILES_NAMES,
+    BasicTokenizer,
+    BertTokenizer,
+    WordpieceTokenizer,
+    _is_control,
+    _is_punctuation,
+    _is_whitespace,
+)
+from transformers.testing_utils import require_tokenizers, slow
+
+from ..test_tokenization_common import TokenizerTesterMixin, filter_non_english
+
+
+@require_tokenizers
+class BertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+
+    tokenizer_class = BertTokenizer
+    rust_tokenizer_class = BertTokenizerFast
```

Comment:
The following changes should be made so that `MobileBERT` classes are used - and not Bert's one.

I also took the opportunity to offer to add a comment explaining where the code was copied from and what changes were applied - - I probably don't have them all! 
```suggestion
# Copied from transformers.tests.bert.test_modeling_bert.py with Bert->MobileBERT 
@require_tokenizers
class MobileBERTTokenizationTest(TokenizerTesterMixin, unittest.TestCase):

    tokenizer_class = MobileBERT
    rust_tokenizer_class = MobileBERTFast
```